docs/components/integrations.rst

```diff
@@ -87,6 +87,10 @@ The ``HelloWorldIntegration`` needs to implement ``\Mautic\IntegrationsBundle\In
         }
     }
 
+.. note:: 
+    `HelloWorldIntegration::NAME` must only contain letters (a-zA-Z).
```

Comment:
```suggestion
   ``HelloWorldIntegration::NAME`` must only contain letters such as a-z or A-Z.
```

Inline code requires double back-ticks for RST, which is why the linter is failing. Also removed the brackets and expanded in text per the prose suggestion.